configure.py

```diff
@@ -0,0 +1,24 @@
+API_VERSION = '2.1'
```

Comment:
what is this for?

Reply:
AMBuild API. AMBuild requires the use of Python as well as AMBuildScript files, manually typed. Also if you want to know more on AMBuild and why it's used, check this out:

https://wiki.alliedmods.net/Ambuild